var/spack/repos/builtin/packages/libxml2/package.py

```diff
@@ -42,8 +43,11 @@ class Libxml2(Package):
 
     def install(self, spec, prefix):
         if '+python' in spec:
-            site_packages_dir = os.path.join(prefix, 'lib/python%s.%s/site-packages' %(spec['python'].version[:2]))
-            python_args = ["--with-python=%s" % spec['python'].prefix, "--with-python-install-dir=%s" % site_packages_dir]
+            site_packages_dir = os.path.join(prefix,
+                                             'lib/python%s/site-packages'
+                                             % (spec['python'].version.up_to(2)))
```

Comment:
sure -- that would be great :).  Would it be clearer if it were set on the python spec?  Basically this:

```python
    spec['python'].site_packages_dir
```

Instead of a mysterious module-scoped variable.